### ISSUE TITLE
Contact avatar menu doesn't show on phone when search bar is used

### ISSUE THREAD

**Issue by AgnesElisa** [friendica/friendica#5126]
### Expected behavior
When on the "all contacts" (or other contact pages) on the phone on frio theme, there should be a menu when you tap a person's avatar

### Actual behavior
When I choose a friend from the list and I tap the avatar, I get the menu like I should. But when I first use the search bar, the avatar then become untappable

### Steps to reproduce the problem
Use your phone, have frio theme, go to "contacts" use the search bar to find a specific contact, tap the avatar.

### Friendica version you encountered the problem
2018.05-rc-1265

**Comment by AlfredSK** [comment]
Yeah, it's the auto-search. If you finish the search by tapping on the "Find" button in the search field it works. But this is a UX issue for sure.
